proxy-node-gateway/src/dist/config.yml

```diff
@@ -31,7 +31,9 @@ replayChecker:
 translatorService:
   url: ${TRANSLATOR_URL}
   client:
+    timeout: ${TRANSLATOR_CONNECTION_TIMEOUT:-5s}
     connectionTimeout: ${TRANSLATOR_CONNECTION_TIMEOUT:-5s}
+    connectionRequestTimeout: ${TRANSLATOR_CONNECTION_TIMEOUT:-5s}
```

Comment:
Where does this get used? I've grepped the proxy-node repo and can only find it mentioned here in the config?

Reply:
The `client:` block gets deserialised into a `JerseyClientConfiguration` which gets provided to the `TranslatorService` that gets built by `TranslatorServiceConfiguration`. The `*timeout:` options are standard params for a Jersey client as per [this Dropwizard class](/dropwizard/dropwizard/blob/master/dropwizard-client/src/main/java/io/dropwizard/client/HttpClientConfiguration.java).